pygeoapi/api.py

```diff
@@ -2759,6 +2759,11 @@ def get_collection_tiles_data(
             return self.get_exception(
                 HTTPStatus.BAD_REQUEST, headers, format_,
                 'InvalidParameterValue', msg)
+        except ProviderInvalidQueryError:
```

Comment:
I think this would even work without this code change. `ProviderInvalidQueryError` is a subclass of `ProviderGenericError`, so it's also caught in the `except` block below, and since `ProviderInvalidQueryError` has set `http_status_code` to 400, this will effectively produce a 400 response